provider/rfc2136/rfc2136.go

```diff
@@ -45,7 +46,7 @@ const (
 type rfc2136Provider struct {
 	provider.BaseProvider
 	nameserver      string
-	zoneName        string
+	zoneName        []string
```

Comment:
```suggestion
	zoneNames        []string
```

Reply:
I thought about it, but it breaks compatibility

Reply:
What about retaining `zoneName` and add `zoneNames`? then append it to the slice like so:
``` go
if zoneName != "" {
    zoneNames = append(zoneNames, zoneName)
}
```

Reply:
I think this is unnecessary, we can rename the field inside the provider, but leave the old flag --zone-name

Reply:
There is changes https://github.com/kubernetes-sigs/external-dns/pull/3976/commits/98e971604422e8d004a21a368941fcf890b6faaf